acbuild/acbuild.go

```diff
@@ -140,7 +144,45 @@ GLOBAL OPTIONS:
 // terminator.
 func runWrapper(cf func(cmd *cobra.Command, args []string) (exit int)) func(cmd *cobra.Command, args []string) {
 	return func(cmd *cobra.Command, args []string) {
+		if aciToModify != "" {
+			contextualCommands := []string{"begin", "end", "abort"}
+			for _, cc := range contextualCommands {
+				command := strings.Split(cmd.Use, " ")[0]
```

Comment:
It's not strange slice semantics here, more just knowing that strings.Split will never return an empty slice. The comments there because if I were in your position and noticed it I'd either comment on it or check Split's implementation. 

> On Oct 13, 2015, at 17:35, Jonathan Boulle <notifications@github.com> wrote:
> 
> In acbuild/acbuild.go:
> 
> > @@ -140,7 +144,45 @@ GLOBAL OPTIONS:
> >  // terminator.
> >  func runWrapper(cf func(cmd *cobra.Command, args []string) (exit int)) func(cmd *cobra.Command, args []string) {
> >  	return func(cmd *cobra.Command, args []string) {
> > +		if aciToModify != "" {
> > +			contextualCommands := []string{"begin", "end", "abort"}
> > +			for _, cc := range contextualCommands {
> > +				command := strings.Split(cmd.Use, " ")[0]
> is that a thinly veiled dig at me? ;-)
> I don't know why I keep un-learning the slice semantics, too many false memories..
> 
> ―
> Reply to this email directly or view it on GitHub.
>